config/routes.rb

```diff
@@ -14,4 +20,5 @@
     end
   end
   resources :relationships, only: [:create, :destroy]
+  resources :categories
```

Comment:
@johnedelmapa
All admin related routes, controller, and pages should be in its own `admin` namespace.
Routes should look like this
- `admin/categories`
- `admin/categories/:id`
- `admin/categories/:id/edit`
...

Read about namespacing here: https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing